package.json

```diff
@@ -53,5 +53,8 @@
     "ts-node": "^10.9.1",
     "typescript": "^4.9.5",
     "winston": "^3.8.2"
+  },
+  "dependencies": {
+    "semver": "^7.5.4"
```

Comment:
Adding a dep is quite a commitment, are we 100% we need this?

Reply:
We use it to compare the rabbitmq management versions, maybe we can try to hardcode the comparison?